src/test/app/AmendmentTable_test.cpp

```diff
@@ -91,19 +110,34 @@ class AmendmentTable_test final : public beast::unit_test::suite
 
     std::unique_ptr<AmendmentTable>
     makeTable(
+        Application& app,
+        std::chrono::seconds majorityTime,
+        Section const supported,
+        Section const enabled,
+        Section const vetoed)
+    {
+        auto t = make_AmendmentTable(
+            app, majorityTime, supported, enabled, vetoed, journal);
+        return t;
```

Comment:
Why not `return make_AmendmentTable...`?

Reply:
Done.